sql/hive/src/main/scala/org/apache/spark/sql/hive/client/package.scala

```diff
@@ -27,7 +27,8 @@ package object client {
   // scalastyle:off
   private[hive] object hive {
     case object v12 extends HiveVersion("0.12.0")
-    case object v13 extends HiveVersion("0.13.1")
+    case object v13 extends HiveVersion("0.13.1",
+      exclusions = Seq("commons-httpclient:commons-httpclient"))
```

Comment:
Workaround: [NOT FOUND  ] commons-httpclient#commons-httpclient;3.0.1!commons-httpclient.jar (1ms)

https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/104649/testReport/org.apache.spark.sql.hive/HiveExternalCatalogVersionsSuite/backward_compatibility/
https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/104649/testReport/org.apache.spark.sql.hive.execution/Hive_2_1_DDLSuite/SPARK_21617__ALTER_TABLE_for_non_compatible_DataSource_tables/

Reply:
It's our jenkins issue, will rever this change.